toqito/channels/tests/test_realignment_invalid_dim.py

```diff
@@ -0,0 +1,23 @@
+import pytest
```

Comment:
We don't have to have a test file that deviates from the naming convention of the file it is testing. This breaks project consistency.